scripts/build.sh

```diff
@@ -53,6 +53,13 @@ if [[ "${TARGET_PLATFORM}" != win-* ]]; then
     EXTRA_CONSTRUCTOR_ARGS="${EXTRA_CONSTRUCTOR_ARGS} --conda-exe ${MICROMAMBA_FILE} --platform ${TARGET_PLATFORM}"
 fi
 
+echo "***** Set virtual package versions *****"
+if [[ "${TARGET_PLATFORM}" == linux-* ]]; then
+    export CONDA_OVERRIDE_GLIBC=2.12
+elif [[ "${TARGET_PLATFORM}" == linux-* ]]; then
+    export CONDA_OVERRIDE_OSX=10.9
```

Comment:
```suggestion
elif [[ "${TARGET_PLATFORM}" == osx-* ]]; then
    export CONDA_OVERRIDE_OSX=10.9
```
I'm fine with that change.
I generally makes sense to but the overrides in.
As for `10.9` concretely, I've also thought about doing that but didn't consider it too be of relevance -- what's the reasoning for going with that still (in light that we changed the default nearly 2 months ago)?

Reply:
It was to see if this propagated. Apparently not.

Reply:
if this does nothing, should we remove it entirely?

Reply:
It does work; there was just a typo in the initial commit (`linux-*` instead of `osx-*`).
Even makes the PyPy builds fail -- haven't looked into why yet.

Reply:
`conda=24.1.1` onward has dep `pypy3.9 >=7.3.15` and that one has been built against 10.13 already.
So, we need target 10.13 for that one.

refs:
https://github.com/conda-forge/pypy3.6-feedstock/pull/116/files#diff-92d4214334555338c6e098e1462e4dec3ca2032a9f2973bf5558d6b8c00503ceR2